kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/ZooKeeperHiveClientHelper.java

```diff
@@ -38,6 +38,9 @@ private static String getZooKeeperNamespace(JdbcConnectionParams connParams) {
     if ((zooKeeperNamespace == null) || (zooKeeperNamespace.isEmpty())) {
       zooKeeperNamespace = JdbcConnectionParams.ZOOKEEPER_DEFAULT_NAMESPACE;
     }
+    if (zooKeeperNamespace.startsWith("/") || zooKeeperNamespace.endsWith("/")) {
+      zooKeeperNamespace.replaceAll("^/+", "").replaceAll("/+$", "");
```

Comment:
the call is cheap, we can remove if to simplify the code, you should re-assign the value to variable `zooKeeperNamespace`, and please add unit tests which should contains both positive and negative cases to verify your change

Reply:
done